tests/testthat/helper-functions.R

```diff
@@ -195,24 +195,13 @@ getStyleValue <- function(html, xpath, style.name) {
 }
 
 ## testthat there is no warning generated by a piece of code.
-gives_no_warning <- function(){
-  function(expr) {
-    warnings <- evaluate_promise(expr)$warnings
-    s <- ifelse(length(warnings)==1, "", "s")
-    expectation(
-      length(warnings) == 0,
-      paste0("created ", length(warnings),
-             " warning", s, ": ",
-             paste(warnings, collapse=", ")),
-      "no warnings given"
-    )
-  }
-}
-expect_no_warning <- function(object, ..., info=NULL, label=NULL){
-  if (is.null(label)) {
-    label <- testthat:::find_expr("object")
-  }
```

Comment:
`testthat` has changed too and `testthat:::find_expr` now [throws an error](https://app.wercker.com/#buildstep/5762874ea79f5ea83002d120). Had to try a different approach for `expect_no_warning`